dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/models/queue.py

```diff
@@ -33,10 +32,3 @@ def __init__(
         description: Optional[str] = "",
     ):
         super().__init__(name, description)
-
-    def create_if_not_exists(self, user=configuration.USER_NAME) -> None:
-        """Create Queue if not exists."""
-        gateway = launch_gateway()
-        # Here we set Queue.name and Queue.queueName same as self.name
-        result = gateway.entry_point.createProject(user, self.name, self.name)
-        gateway_result_checker(result, None)
```

Comment:
Do you forget adding some method for this?

Reply:
Because I didn't find the createProject method in the [PythonGateway.java ](https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/python/PythonGateway.java).

And the [Queue](https://github.com/apache/dolphinscheduler/blob/dev/dolphinscheduler-python/pydolphinscheduler/src/pydolphinscheduler/models/queue.py) class doesn't seem to be in use .

So I deleted that code

Reply:
Got it